src/websys_bridge.rs

```diff
@@ -164,25 +185,37 @@ pub fn make_websys_el<Ms>(el_vdom: &mut El<Ms>, document: &web_sys::Document) ->
         set_style(&el_ws, &el_vdom.style)
     }
 
-    // Don't attach listeners here,
     el_ws.into()
 }
 
-/// Similar to `attach_el_and_children`, but assumes we've already attached the parent.
+/// Similar to attach_el_and_children, but for text nodes
+pub fn attach_text_node(text: &mut Text, parent: &web_sys::Node) {
+    let node_ws = text.node_ws.take().expect("Missing websys node for Text");
+    parent
+        .append_child(&node_ws)
+        .expect("Problem appending text node");
+    text.node_ws.replace(node_ws);
+}
+
+/// Similar to attach_el_and_children, but without attaching the elemnt. Useful for
+/// patching, where we want to insert the element at a specific place.
 pub fn attach_children<Ms, Mdl, ElC: ElContainer<Ms>>(
     el_vdom: &mut El<Ms>,
-    app: &App<Ms, Mdl, ElC>,
+    app: &App<Ms, Mdl, ElC>, // To avoid inferring type for Mdl.
```

Comment:
> // To avoid inferring type for Mdl.

What does it mean?

Reply:
I'm not sure what's going on, but without passing this, we get compile errors about needing type annotations on `Ms`.

Reply:
Well, I tried to remove `app` and it seems to work.. https://github.com/David-OConnor/seed/pull/177

Reply:
**RESOLVED**